inscriptions/ISic1529.xml

```diff
@@ -170,11 +175,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/>Κα<hi rend="stacked">λα</hi><supplied reason="omitted">ν</supplied><g type="staurogram">&#65952;</g>δ<hi rend="ligature">ίω</hi><hi rend="ligature">νο</hi>ς
+                    <lb n="2"/>τόπος
```

Comment:
I wonder is we should put the whole staurogram in line 1a. It breaks not just kalandon but topos as well, yet we can't represent all that in the text.

Reply:
maybe, as usual, we could put the staurogram where it 'begins' (line 1, before/after delta) and use the layout desc to explain the whole arrangement...